CCPAPI/PAPI/ccPApi.cpp

```diff
@@ -0,0 +1,420 @@
+//##########################################################################
+//#                                                                        #
+//#                               CCPAPI                                   #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU Library General Public License as       #
+//#  published by the Free Software Foundation; version 2 or later of the  #
+//#  License.                                                              #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+//#                                                                        #
+//##########################################################################
+
+#include "ccPApi.h"
+
+#include <GeometricalAnalysisTools.h>
+
+//qCC_db
+#include <ccHObjectCaster.h>
+//#include <ccColorScalesManager.h>
```

Comment:
Suggest removing commented-out things (here and throughout). If they are incomplete, then at least comment that they aren't working yet.